packages/react-core/src/components/DataList/examples/DataList.md

```diff
@@ -13,8 +13,7 @@ propComponents:
     'DataListItemRow',
     'DataListToggle',
     'DataListContent',
-    'DataListDragButton',
-    'DataListControl'
```

Comment:
just double checking - we could remove this, but we could either:
- additionally link to the new dragdrop docs to show the new recommended way to implement the draggable datalist (since that example now lives in DragDrop rather than datalist)
- OR keep these examples here, mark them deprecated, and possibly still link to the new recommended solution in the DragDrop examples.

Reply:
The same would be true for the dual list selector

Reply:
Added back the examples as the old dragDrop is still available, and added a note pointing to the new implementation.